src/components/shared/Story/index.tsx

```diff
@@ -58,8 +68,61 @@ const CallToAction: React.StatelessComponent<{
               </TextLink>
             );
           case CallToActionDataType.TIP:
-            // TODO(tina): Make real component
-            return <Text key={idx}>{piece.data.tipTotal} as tip!</Text>;
+            return (
+              <View
```

Comment:
maybe worth putting into it's own component?

Reply:
I'll do this in a later PR